pom.xml

```diff
@@ -51,6 +51,8 @@
 ]]>
     </checkstyle.header.template>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
```

Comment:
@bsremac It seems you found an issue with checkstyle plugin configuration in kiegroup projects (declaring `checkstyle.logViolationsToConsole` should not have any effect). I'd recommend removing that property from configuration altogether, as the default value of the "correct" `checkstyle.console` property [is true](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#logViolationsToConsole) anyway